### PR TITLE
Project

### PR DESCRIPTION
**Pull request by SirRedrick**

### CODE REVIEW

File: src/components/Card.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from './Card.module.css';
+import mobileImage from '../images/illustration-woman-online-mobile.svg';
+import desktopImage from '../images/illustration-box-desktop.svg';
+
+export default function Card({ title, children }) {
```
Comment (v1nn1k) [inline src/components/Card.js:6]: I'd add the PropTypes here

File: src/components/Card.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	return (
+		<div className={styles.card}>
+			<div className={styles.image_container}>
+				<img className={styles.mobile_image} src={mobileImage} alt="" />
```
Comment (v1nn1k) [inline src/components/Card.js:10]: Respect alt text)

File: src/components/FaqList.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import styles from './FaqList.module.css';
+import arrowDown from '../images/icon-arrow-down.svg';
+
+export default function FaqList({ data }) {
+	const [opened, setOpened] = useState(null);
```
Comment (v1nn1k) [inline src/components/FaqList.js:6]: I think it's better to be explicit and say `openedIndex, setOpenedIndex`

File: src/components/FaqList.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+	const open = (index) => {
+		if (opened === index) return setOpened(null);
```
Comment (v1nn1k) [inline src/components/FaqList.js:9]: use full block
```
if(something) {
  ...
}

File: src/components/FaqList.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+		setOpened(index);
+	};
+
+	return (
+		<ul className={styles.list}>
+			{data.map((entry, index) => (
+				<li key={index}>
```
Comment (v1nn1k) [inline src/components/FaqList.js:16]: Don't use array index as key

Comment (v1nn1k) [inline src/components/FaqList.js:16]: add local `const isOpened = opened === index` variable and use later on

**Review by v1nn1k — CHANGES_REQUESTED** [review]
Hi! Great one, don't forget to check out some other projects reviews ;)

**Review by olehmell — APPROVED** [review]
Good start. I have nothing to add after @v1nn1k! Good luck learning!

File: src/components/FaqList.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+					<div className={styles.item_header}>
+						<h2 onClick={() => open(index)} className={opened === index ? styles.active : null}>
+							{entry.question}
+						</h2>
+						<img className={opened === index ? styles.opened : null} src={arrowDown} alt="" />
+					</div>
+
+					{opened === index ? <p>{entry.answer}</p> : null}
```
Comment (kyrylo-burmelov) [inline src/components/FaqList.js:24]: It's better to use && instead of ? in this case

**Comment by SirRedrick** [comment]
Applied all of the suggestions listed in the reviews, would be really grateful if you could approve the changes

**Comment by v1nn1k** [comment]
> Applied all of the suggestions listed in the reviews, would be really grateful if you could approve the changes

Great, I’ll try to take a look sometime later

**Pull request closed without merging**